From b6fafa063412f16658732ec4ab2b238512076b60 Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Thu, 5 Jul 2012 11:20:51 +0100
Subject: [PATCH] configs: vexpress: Extra options needed for TC2

TC2 has 5 CPUS and 2GB DRAM so allow the kernel to make use of all of
these resources.

Signed-off-by: Jon Medhurst <tixy@linaro.org>
(cherry picked from commit 1d48350f734ab382da2f778311991bb266147562)
Signed-off-by: Alex Shi <alex.shi@linaro.org>
---
 linaro/configs/vexpress.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/linaro/configs/vexpress.conf b/linaro/configs/vexpress.conf
index 6576492..a75400a 100644
--- a/linaro/configs/vexpress.conf
+++ b/linaro/configs/vexpress.conf
@@ -1,6 +1,9 @@
 CONFIG_ARCH_VEXPRESS=y
 CONFIG_ARCH_VEXPRESS_CA9X4=y
 CONFIG_HAVE_ARM_ARCH_TIMER=y
+CONFIG_NR_CPUS=8
+CONFIG_HIGHMEM=y
+CONFIG_HIGHPTE=y
 CONFIG_CMDLINE="console=ttyAMA0,38400n8 root=/dev/mmcblk0p2 rootwait mmci.fmax=4000000"
 CONFIG_VFP=y
 CONFIG_NEON=y
-- 
1.8.0

